tests/data/scan_folder/scan_sub_folder/sub_folder_test.txt

```diff
@@ -0,0 +1,3 @@
+not matched
+ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=
```

Comment:
This requires a new signature to be added to `pyproject.toml`. I'd recommend installing pre-commit, if you haven't already, so that issues such as these are caught before pushing to GH.